arch/arm/boot/dts/bcm2708-rpi-b-plus.dts

```diff
@@ -22,6 +22,11 @@
 		brcm,function = <1>; /* output */
 	};
 
+	i2c0_pins: i2c0 {
+		brcm,pins = <0 1>;
+		brcm,function = <4>;
+	};
+
```

Comment:
We have
```
			i2c0_gpio0: i2c0_gpio0 {
				brcm,pins = <0 1>;
				brcm,function = <BCM2835_FSEL_ALT0>;
			};
```
in bcm283x.dtsi. Any reason to duplicate?

Reply:
Because existing overlays target `i2c0_pins`.